src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
   </head>
   <body>
```

Comment:
Use 2 spaces for indentation in your file (not a tab character) to make sure your formatting will look the same everywhere. The indentation before the `<body>` tag seems to be more than 2 spaces.